R/McmcOptions-class.R

```diff
@@ -14,9 +14,20 @@ NULL
 #' @slot step (`count`)\cr only every `step`-th iteration is saved after
 #'   the `burnin`. In other words, a sample from iteration
 #'   `i = 1,...,iterations`, is saved if and only if
-#'   `(i - burnin) mod step = 0`. For example, for `iterations = 6`, 
-#'   `burnin = 0` and `step = 2`, only samples from iterations `2,4,6`
-#'   will be saved.
+#'   `(i - burnin) mod step = 0`.\cr
+#'   For example, for `iterations = 6`, `burnin = 0` and `step = 2`, only
+#'   samples from iterations `2,4,6` will be saved.
+#' @slot RNG (`list`)\cr a Random Number Generator (RNG) type and seed used
+#'   by `rJAGS`. It must be a list with two elements named `.RNG.name` and
+#'   `.RNG.seed`.\cr
+#'   The `.RNG.name` is the the name of the RNG type and it is one out of the
+#'   following four values: `base::Wichmann-Hill`, `base::Marsaglia-Multicarry`,
```

Comment:
I think the user should be able to select from character vector without the `base::` prefix. that is an internal implementation detail we can handle ourselves

Reply:
Sure, Thanks! I now did it so that it is pasted in the user constructor, so the slots keep the prefix. This is ok, but it has one drawback - the error message from the validity, about wrong prefix....Maybe would be better to keep the names without prefix in slots too, and then have just paste the prefix in `mcmc`...